4001-5000/LIT4867Theodore.xml

```diff
@@ -63,18 +66,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:P129_is_about" active="LIT4867Theodore" passive="PRS9475Theodoru"/>
                <relation name="saws:isAttributedToAuthor" active="LIT4867Theodore" passive="PRS5720JohnChr"/>
+               <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6463Letter" passive="LIT1151Asceti"/>
             </listRelation>
             <listBibl type="secondary">
                <bibl>
                   <ptr target="bm:Lusini1988patristica"/>
                   <citedRange>481-482</citedRange>
                </bibl>
                <bibl>
-                  <ptr target="bm:Bausi2007John"/>
+                  <ptr target="bm:Witakowski2007JohnChry"/>
```

Comment:
is the old bibliographical reference wrong?

Reply:
Yes, checked in EAe, the old one referred to John the Baptist, not Chrysostom.